test/3nodetest/Makefile

```diff
@@ -14,7 +14,7 @@ data/filetiny: Makefile
 	cp Makefile ./data/filetiny # simple
 
 data/filerand: bin/random
-	./bin/random 50000000 > ./data/filerand
+	./bin/random 50MB > ./data/filerand
```

Comment:
that or seeding random, which i doubt would cause problems in this scenario -- seeding might also make jenkins runs faster (if it reuses dir, which it may not...)